pandas/tests/series/indexing/test_indexing.py

```diff
@@ -853,6 +853,17 @@ def test_setitem_slice_into_readonly_backing_data():
     assert not array.any()
 
 
+def test_access_none_value_in_multiindex():
+    # GH34318: test that you can access a None value using .loc through a Multiindex
+
+    expected = None
+    result = pd.Series([None], pd.MultiIndex.from_arrays([["Level1"], ["Level2"]])).loc[
```

Comment:
can you write this in a more readable way

```
s = Series(......)
result = s.loc....
assert result is None
```

Reply:
pls add all the examples here as well: https://github.com/pandas-dev/pandas/issues/34318#issuecomment-632834528